coderd/notifications/testdata/rendered-templates/webhook/TemplateWorkspaceAutobuildFailed.json.golden

```diff
@@ -19,7 +19,8 @@
       "name": "bobby-workspace",
       "reason": "autostart"
     },
-    "data": null
+    "data": null,
+    "targets": null
```

Comment:
I guess we're never testing with targets?

Reply:
def not as much as we could - I added one on the logic but we can improve the golden files.

Reply:
Which golden file did that update? I don't see it.

Reply:
Ok I took a bit longer than expected , but this targets never changes - it is part of [the buildPayload](https://github.com/coder/coder/blob/77479cdd51f5154054e27734b30900a01f014729/coderd/notifications/enqueuer.go#L127) which takes directly data fetched from db that does not contain the targets.

I'd like to clean it up as a follow-up PR, seems like there's some mixed fields with targets - one within the `payload` and another one outside of it.